Add paidBy to get payment requests by id (#284)
centrapay/centrapay-docs: src/api/payment-requests/payment-requests.md

attrubutes to be supplied. In the case of GS1 product classifications this
corresponds to the GPC brick attributes.

### Paid By

> The Paid By provides a summary of the aggregated transactions by asset type for the Payment Request.

Reviewer: ```suggestion
The Paid By provides a summary of the transactions after the Payment Request was paid.
```

{% h4 Mandatory Fields %}

|     Field      |        Type        |                           Description                            |
| -------------- | ------------------ | ---------------------------------------------------------------- |

> | assetTotals    | Array              | The [Asset Totals](#asset-total) indicating a transaction on the payment request. |

Reviewer: ```suggestion
| assetTotals    | Array              | The sum of paid amounts [Asset Totals](#asset-total) for each transacted asset type. |
```
